test/performance/search/views/view_kmer_hash_benchmark.cpp

```diff
@@ -104,8 +103,22 @@ static void naive_kmer_hash(benchmark::State & state)
 
     for (auto _ : state)
     {
-        for (auto h : seq | seqan3::views::naive_kmer_hash(k))
+        // creates a initial subrange covering exactly 'k-1' characters
+        auto subRangeBegin = begin(seq);
```

Comment:
naming should be `subrange_begin`, `subrange_end`, I think

Reply:
Yes! much better!